client/src/components/SolutionViewer/SolutionViewer.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { Panel } from '@freecodecamp/react-bootstrap';
+import Prism from 'prismjs';
+import React from 'react';
+
+interface solutionviewerProps {
+  challengeFiles: [];
+  solution: string;
+}
+
+function SolutionViewer({
+  challengeFiles,
+  solution = '// The solution is not available for this project'
+}: solutionviewerProps): JSX.Element {
+  return (
+    <>
+      {challengeFiles?.length ? (
+        challengeFiles.map((challengeFile: string) => (
```

Comment:
```suggestion
        challengeFiles.map((challengeFile: ChallengeFile) => (
```
This is not a string.